view/components/main.html

```diff
@@ -56,7 +56,7 @@
                 </div>
                 <div id="queried-data" data-ng-if="message.result.length !== 0">
                     <br>
-                    <div data-ng-repeat="objectQuery in message.result">
+                    <div data-ng-repeat="objectQuery in message.result" id="queried-info">
```

Comment:
really sad, that there is no $index for the index count, as indicated in the description picture.